s-sql/s-sql.lisp

```diff
@@ -189,7 +194,7 @@ characters other than #\-)"
 (deftype serial8 () 'integer)
 
 (deftype db-null ()
-  "Type for representing NULL values. Use like \(or integer db-null)
+  ")Type for representing NULL values. Use like \(or integer db-null)
```

Comment:
There's a superfluous ")" at the beginning of the docstring.

Reply:
Ooops, you are right!